[I2C] Add I2C frequency setting

TD-er opened the pull request:
For I2C UART an I2C clock of 400 kHz is needed to be able to handle continuous streams of data.
But the default was still 100 kHz, unless you selected a plugin that may have set the I2C frequency to 400 kHz like the Framed OLED plugin.

TD-er committed:
[I2C] Add I2C frequency setting

For I2C UART an I2C clock of 400 kHz is needed to be able to handle continuous streams of data.
But the default was still 100 kHz, unless you selected a plugin that may have set the I2C frequency to 400 kHz like the Framed OLED plugin.

TD-er committed:
ESPEasy Status LED is not working on ESP32 with buildin LED (#2940)

Fixes: #2940

Copy-paste fix from @thomastech https://github.com/letscontrolit/ESPEasy/issues/2940#issuecomment-599627656

uzi18 commented:
@TD-er be aware, some I2C devices does not support higher frequencies, not sure if we are affected but there is a margin about 400kHz

TD-er commented:
The current implementation of I2C in the Arduino core does limit the max. frequency to 400 kHz. (when running at 80 MHz)
The Framed OLED plugin already tried to set it to 700 kHz and that doesn't seem to cause issues.

The I2C UART chip I'm testing with here explicitly has a max. frequency of 400 kHz and I tried to set it higher but wasn't able to. 
I have not tested with the scope by the way, but it kept on working and the logged times in the timingstats did not change, while they did improve upto 400 kHz.

TD-er committed:
Pulse Counter must retain its values after reboot (#2268)

Fixes #2268

TD-er committed:
[Build] Disable test_ESP8266_4M1M_VCC_MDNS_SD due to max sketch size

Temporary disabled so we can make a nightly build

TD-er committed:
Fix resetpulsecounter for P003_Pulse (#2710)

Fixes: #2710

I really need to make sense of the argument positions as this is really error prone as shown here.

TD-er committed:
[Blynk] Fix building C015 Blynk controller

As found [on the forum](https://www.letscontrolit.com/forum/viewtopic.php?f=6&t=7538)

TD-er committed:
[JSON] Fix JSON output of values for single task

Generated JSON had trailing TTL value which rendered it invalid JSON.
